patch/SwapConfTexts.scala

```diff
@@ -0,0 +1,174 @@
+//package utils //put right name
```

Comment:
I think it would be better to use a manual database update script instead of this patch file and put `Deprecated` notes on top of all columns in `Conference.scala` that are no longer used.